### PR TITLE
Promote @astencel-sumo to Maintainer

### PR DESCRIPTION
**Pull request by dmitryax**
Andrzej contributed a lot to the repository. The maintainers propose to promote Andrzej to Maintainer.

### CODE REVIEW

**Review by djaglowski — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Review by songy23 — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Comment by dmitryax** [comment]
Waiting for approval from @astencel-sumo before merging

**Review by atoulme — APPROVED** [review]

**Review by andrzej-stencel — APPROVED** [review]
I'm honored, thank you for the trust. I'll do my best.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**
